<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crea Mod Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="installButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modFile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>512</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="modFileButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mod file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Crea path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="creaPath">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="creaPathButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uninstallButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Uninstall</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Installed Mods</string>
    </property>
   </widget>
   <widget class="ModListView" name="installedMods">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>30</y>
      <width>201</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModListView</class>
   <extends>QListView</extends>
   <header>ModListView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
